### PR TITLE
git-hound 0.6.2 (new formula)

### PR DESCRIPTION
**Pull request by ezekg**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by ruurd — COMMENTED** [review]
Shipit!

**Comment by dawidd6** [comment]
Latest release (and commit) has been made almost 3 years ago. Is this project still maintained?

**Comment by ezekg** [comment]
> Latest release (and commit) has been made almost 3 years ago. Is this project still maintained?

I'm the maintainer. It's stable.

**Comment by dawidd6** [comment]
Could you migrate it to Go Modules, make a new release and update this PR?

File: Formula/git-hound.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+    houndpath = buildpath/"src/github.com/ezekg/git-hound"
+    houndpath.install buildpath.children
+
+    cd houndpath do
+      system "glide", "install"
+      system "go", "build", "-o", "git-hound", "-ldflags", "-X main.version=#{version}"
+      bin.install "git-hound"
+      prefix.install_metafiles
```
Comment (dawidd6) [inline Formula/git-hound.rb:20]: This line is not needed.

**Comment by ezekg** [comment]
Closing in favor of https://github.com/Homebrew/homebrew-core/pull/61764.

**Pull request closed without merging**